client/go/internal/cli/cmd/visit.go

```diff
@@ -133,7 +134,7 @@ $ vespa visit --field-set "[id]" # list document IDs
 	cmd.Flags().StringVar(&vArgs.to, "to", "", `Timestamp to visit up to, in seconds`)
 	cmd.Flags().IntVar(&vArgs.sliceId, "slice-id", -1, `The number of the slice this visit invocation should fetch`)
 	cmd.Flags().IntVar(&vArgs.slices, "slices", -1, `Split the document corpus into this number of independent slices`)
-	cmd.Flags().StringVar(&vArgs.bucketSpace, "bucket-space", "default", `"default" or "global" bucket space`)
+	cmd.Flags().StringSliceVar(&vArgs.bucketSpaces, "bucket-space", []string{"global", "default"}, `"default" or "global" bucket space`)
```

Comment:
... or "default,global" ?

Reply:
my thinking was that "global" should be visited first.

Reply:
ok. I am just confused, there are three options "global", "default" and "global,default", right? (and the last one is default). I think the help text should reflect that